### PR TITLE
Team Teaching - Render stats work with engine

### PR DESCRIPTION
**Pull request by Atlante45**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: a4340b17e964e45ea4cbc9c4b0b59f00d797d07d available at: https://gist.github.com/8a1176699598201fd081<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5044.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5044.dmg

File: interface/src/ui/Stats.cpp [hunk 1]
```diff
@@ -467,31 +467,16 @@ void Stats::display(
     horizontalOffset += 5;
 
     // Model/Entity render details
-    EntityTreeRenderer* entities = Application::getInstance()->getEntities();
     octreeStats.str("");
-    octreeStats << "Entity Items rendered: " << entities->getItemsRendered() 
-                << " / Out of view:" << entities->getItemsOutOfView()
-                << " / Too small:" << entities->getItemsTooSmall();
+    octreeStats << "Triangles: " << _renderDetails._trianglesRendered
```
Comment (ZappoMan) [inline interface/src/ui/Stats.cpp]: I still think we want to have the out of view and too small stats so we can see that culling and LOD are working...

File: libraries/render/src/render/DrawTask.cpp [hunk 2]
```diff
@@ -80,15 +79,9 @@ void render::cullItems(const SceneContextPointer& sceneContext, const RenderCont
         bool outOfView = args->_viewFrustum->boxInFrustum(bound) == ViewFrustum::OUTSIDE;
         if (!outOfView) {
             bool bigEnoughToRender = (args->_shouldRender) ? args->_shouldRender(args, bound) : true;
-            
             if (bigEnoughToRender) {
                 outItems.push_back(id); // One more Item to render
-                args->_itemsRendered++;
```
Comment (ZappoMan) [inline libraries/render/src/render/DrawTask.cpp]: I think we need to keep these... this is a good place to do this

**Comment by hifi-gustavo** [comment]
Build console output for commit: 94ba84a84c51bdf39c1ceb7c427d998168e2f2c5 available at: https://gist.github.com/8a1176699598201fd081<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5044.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5044.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: f19c0d8eecb589ca3c88753bd36ff63f6e826883 available at: https://gist.github.com/8a1176699598201fd081<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5044.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5044.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 5f16b8752b53ae432c3658ea1781617e0689bbe1 available at: https://gist.github.com/8a1176699598201fd081<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5044.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5044.dmg

**Pull request merged**